recipes/tardis/meta.yaml

```
@@ -0,0 +1,107 @@
+{% set name = "tardis-sn" %}
+{% set version = "2.0.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  git_url: https://github.com/tardis-sn/tardis.git
+  git_branch: master
+
+build:
+  number: 0
+  skip: true  # [not win]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - setuptools
+    - python=3
+    - numpy=1.15
+    - scipy=1.1
+    - pandas=0.24
+    - astropy=3
+    - numexpr
+    - Cython=0.29
+    - networkx
+    - pyyaml
+    - jsonschema
+    - pyne=0.5
+    - pytables
+    - h5py
+    - jupyter
+    - notebook
+    - matplotlib
+    - graphviz
+    - pygraphviz
+    - sphinx
+    - nbconvert
+    - numpydoc
+    - docutils
+    - nbformat
+    - nbsphinx
+    - sphinx_bootstrap_theme
+    - sphinxcontrib-bibtex
+    - sphinx_rtd_theme
+    - git-lfs
+    - pytest
+    - pytest-html
+    - requests
+    - coverage
+    - docopt
+    - pytest-cov
+    - codecov
+    - backports
+
+  run:
+    - python=3
+    - numpy=1.15
+    - scipy=1.1
+    - pandas=0.24
```

These should probably be minimum versions with a space after the package name:
```suggestion
    - pandas >=0.24
```

---

Ok!.